examples/15-managed-nodes.yaml

```diff
@@ -15,6 +15,8 @@ managedNodeGroups:
     maxSize: 4
     availabilityZones: ["us-west-2a", "us-west-2b"]
     volumeSize: 20
+    updateConfig:
+      maxUnavailable: 20 # or maxUnavailablePercentage if specified in percentage
```

Comment:
is it `20%` or `20` when setting percentages? Might be worth updating the comment to be clear :smile:

Reply:
true! thanks I'll reword :)

Reply:
a note: maxUnavailable can't be greater than maxSize of the node group

Reply:
thanks for reviewing @ravisinha0506, this has been updated (this code snippet is outdated)!